<ui version="4.0" >
 <class>JobPlotControlsForm</class>
 <widget class="QWidget" name="JobPlotControlsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="14" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>236</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="fileCheck" >
         <property name="text" >
          <string>File Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="byteCheck" >
         <property name="text" >
          <string>Byte Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="13" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>82</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>Refresh</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >:/images/view-refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="1" >
    <widget class="QComboBox" name="fileSetComboBox" />
   </item>
   <item row="10" column="1" >
    <widget class="QComboBox" name="purgedComboBox" />
   </item>
   <item row="8" column="1" >
    <widget class="QComboBox" name="volumeComboBox" />
   </item>
   <item row="7" column="1" >
    <widget class="QComboBox" name="clientComboBox" />
   </item>
   <item row="11" column="1" >
    <widget class="QComboBox" name="levelComboBox" />
   </item>
   <item row="12" column="1" >
    <widget class="QComboBox" name="statusComboBox" />
   </item>
   <item row="6" column="1" >
    <widget class="QComboBox" name="jobComboBox" />
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="plotTypeCombo" />
   </item>
   <item row="5" column="1" >
    <widget class="QSpinBox" name="daysSpinBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="singleStep" >
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSpinBox" name="limitSpinBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum" >
      <number>10000</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="singleStep" >
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="fileSymbolTypeCombo" />
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="byteSymbolTypeCombo" />
   </item>
   <item row="12" column="0" >
    <widget class="QLabel" name="statusLabel" >
     <property name="text" >
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" >
    <widget class="QLabel" name="levelLabel" >
     <property name="text" >
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QLabel" name="purgedLabel" >
     <property name="text" >
      <string>Purged</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="fileSetLabel" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>FileSet</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="volumeLabel" >
     <property name="text" >
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="clientsLabel" >
     <property name="text" >
      <string>Client</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="jobLabel" >
     <property name="text" >
      <string>Job</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QCheckBox" name="daysCheckBox" >
     <property name="text" >
      <string>Days Limit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="limitCheckBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Record Limit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Byte Symbol Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>File Symbol Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Graph Type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
